src/include/bitstring_inline.h

```diff
@@ -370,3 +370,49 @@ __bit_setv(uint8_t *bitf, uint64_t entry, uint8_t width, uint8_t value)
 		break;
 	}
 }
+
+/*
+ * __bit_clear_end --
+ *     Clear the leftover end bits of a fixed-length column store bitstring.
```

Comment:
If this is FLCS specific, it shouldn't be in this file.

Reply:
It's not, in the sense that it's an operation on a bitstring. Can change the comment text (was written to match some of the others in the file...)

Reply:
You're right, I'm wrong -- objection retracted.